make/mt-os/mt-esp32.mk

```diff
@@ -102,6 +102,14 @@ endif
 # mt-tls-mbedtls-esp32.mk. Or delete the line below to turn off TLS lib cross compilation.
 include make/mt-config/mt-tls-$(XI_BSP_TLS)-esp32.mk
 
+ifeq ($(XI_BSP_TLS),wolfssl)
+# WolfSSL TLS BSP configuration
+XI_CONFIG_FLAGS += -DXI_PROVIDE_WOLFSSL_SEED_GENERATOR
+XI_CONFIG_FLAGS += -DXI_PROVIDE_WOLFSSL_XTIME_XGMTIME
```

Comment:
I git cherry-picked this code from my branch, and these config flags slipped through the cracks.

They'll make more sense when you see the rest of the ESP32 PR.